src/components/marginals/Footer/AboutBox.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Heading3 } from '../..';
+import {
+  FirstAboutContainer,
+  FirstRow,
+  FrItems,
+  AboutBox,
+  Para,
+  Content,
+  LogoCont,
+  LogoImg,
+} from './styles';
+
+const AboutContainer = ({ AbtLogo, about_content }) => {
+  console.log(about_content);
+  return (
+    <FirstAboutContainer>
+      <FirstRow>
+        <FrItems>About US</FrItems>
```

Comment:
Follow DRY principle as much as possible.
In cases like these, where you have to use the same component multiple times, run it through a map and make the component dynamic with props.

Reply:
made changes. [commited](https://github.com/dscnitrourkela/project-oregano/pull/241/commits/d0620ff74ed2966e2f08b672b905aa4df4b88997)